handlers/update_resource.go

```diff
@@ -58,9 +58,9 @@ func (d *updateResourceEntry) persistableResourceFromParams(resourceID string, p
 	resource.AtContext = *params.Payload.AtContext
 	resource.AtType = *params.Payload.AtType
 	resource.Label = *params.Payload.Label
-	resource.Preserve = *params.Payload.Preserve
-	resource.Publish = *params.Payload.Publish
-	resource.SourceID = params.Payload.SourceID
+	resource.Administrative = *params.Payload.Administrative
+	resource.Identification = *params.Payload.Identification
+	// resource.Publish = *params.Payload.Publish
```

Comment:
I don't see where Publish is defined anymore. Did it get lost?

Reply:
It's not part of the `Resource` MAP, since that is just what is required for processing across all types. It is in the type-specific MAPs.

Reply:
Cool. I understand.